internal/gazelle.bash.in

```diff
@@ -76,6 +77,20 @@ if [ -z "$gazelle_path" ]; then
   echo "error: could not locate gazelle binary" >&2
   exit 1
 fi
+if [[ -n "${TEST_WORKSPACE+x}" && -n "$WORKSPACE" ]]; then
+  # references https://github.com/bazelbuild/buildtools/blob/1429e15ae755a6762d0edf9198062dc6ed04408d/buildifier/runner.bash.template#L17-L26
+  # use `realpath` if available and `readlink` otherwise (typically macOS)
+  if command -v realpath &> /dev/null; then
```

Comment:
If readlink is correct on Mac (I think it is), it's correct on Linux as well. Let's use it unconditionally.

Reply:
Readlink is a compatible practice, which actually only serves macOS < 13.x. If possible, it should be `readlink -f` or `readpath`.

Reply:
ping @fmeum

Reply:
But why do we need to resolve more than one layer of symlinks? I think that peeling off a single layer is actually both correct and compatible with all platforms (just always use `readlink`).